### PR TITLE
Add Collision between projectiles and tilemap

### PR DESCRIPTION
**Pull request by Jakllp**
#12

### CODE REVIEW

File: resources/overlap/hitbox.gd [hunk 1]
```diff
@@ -3,3 +3,7 @@ extends Area2D
 class_name Hitbox
 
 @export var damage: int = 0
+
+func _on_body_entered(body):
+	if body is TileMap:
```
Comment (Thyrexx) [inline resources/overlap/hitbox.gd:8]: Souldnt be done in this script because it would apply to all hitboxes i.e. enemies as well.

**Review by Thyrexx — CHANGES_REQUESTED** [review]
See comment. change that!

File: resources/overlap/hitbox.gd (hunk 1 above)
Comment (Jakllp) [reply resources/overlap/hitbox.gd:8]: Ok

**Review by Thyrexx — APPROVED** [review]
Nice! Well done!!!

**Pull request merged**